squad/frontend/tests.py

```diff
@@ -46,18 +46,31 @@ def __init__(self):
         self.paginator = self
         self.paginator.num_pages = 0
         self.number = 0
+        self.all_tests = []
+
+    def __get_all_tests__(self, build, search):
+        self.all_tests = []
```

Comment:
duplicates what's already in the constructor

Reply:
fixed